manifests/init.pp

```diff
@@ -349,7 +365,7 @@
     group   => $config_file_group,
     mode    => $config_file_mode,
     content => template('ntp/ntp.conf.erb'),
-    require => Package[$package_name_real],
+    require => $ntp_conf_require,
```

Comment:
Change the package to use `before` then ntp_conf will not need to optionally set this require.